pkg/genlib/config/config.go

```diff
@@ -22,14 +28,46 @@ type Config struct {
 
 type ConfigField struct {
 	Name        string   `config:"name"`
-	Fuzziness   Ratio    `config:"fuzziness"`
+	Fuzziness   float64  `config:"fuzziness"`
 	Range       Range    `config:"range"`
 	Cardinality Ratio    `config:"cardinality"`
 	Enum        []string `config:"enum"`
 	ObjectKeys  []string `config:"object_keys"`
 	Value       any      `config:"value"`
 }
 
+func (r Range) MinAsInt64() (int64, error) {
+	if r.Min == nil {
```

Comment:
we want to differ when `Min/Max` are explicitly set to zero value or are not set at all.
The only was is to make the value a pointer, so if not set the zero value of the pointer will be `nil` and we can differentiate from explicitly being set to the zero value of the underlying pointer type